### PR TITLE
Add an index page to the docu and change readme of the repo.

### PR DESCRIPTION
**Pull request by DunklesArchipel**
Checklist
* [x] Added an entry in `doc/news/`. 
* [x] Added a test for this change. -> not required
* [x] Adapted the documentation for this change. -> It's all about docu

Readme and index are very similar, except that the index is executed during generation, hence showing cell output.

Made some minor modifications to the database documentation.

### CODE REVIEW

**Comment by saraedum** [comment]
There are conflicts that need to be resolved.

File: README.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.6502901.svg)](https://doi.org/10.5281/zenodo.6502901)
 
 The [echemdb repository](https://echemdb.github.io/website/) contains high
 quality experimental and theoretical data on electrochemical systems.
 
-This module provides a Python library to interact with the data in the repository.
+This echemdb module provides a Python library to interact with a database of 
+[frictionless datapackages](https://frictionlessdata.io/)
+containing electrochemical data following [echemdb's metadata schema](https://github.com/echemdb/metadata-schema).
+Such a database can be generated from the data schown on [echemdb.org](https://www.echemdb.org)
```
Comment (saraedum) [inline README.md]:
```suggestion
Such a database can be generated from the data on [echemdb.org](https://www.echemdb.org) 
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 
 The [echemdb repository](https://echemdb.github.io/website/) contains high
 quality experimental and theoretical data on electrochemical systems.
 
-This module provides a Python library to interact with the data in the repository.
+This echemdb module provides a Python library to interact with a database of 
+[frictionless datapackages](https://frictionlessdata.io/)
+containing electrochemical data following [echemdb's metadata schema](https://github.com/echemdb/metadata-schema).
+Such a database can be generated from the data schown on [echemdb.org](https://www.echemdb.org) 
+or it can be created from local files which have the same file structure.
```
Comment (saraedum) [inline README.md]:
```suggestion
or from local files.
```

File: README.md [hunk 3]
```diff
@@ -62,24 +63,21 @@ A single entry can be retrieved with the identifiers provided on the website
 >>> entry = db['engstfeld_2018_polycrystalline_17743_f4b_1']
 ```
 
-Each entry has information about its source
+Each entry has a set of descriptors which contain information about its `source`, the electrochemical system `system`, or the original publication's figure properties `entry.figure_description`.
```
Comment (saraedum) [inline README.md]:
```suggestion
Each entry has a set of descriptors such as its ``source`` or the electrochemical ``system``.
```

File: doc/index.md [hunk 4]
```diff
@@ -14,20 +14,93 @@ kernelspec:
...
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.6502901.svg)](https://doi.org/10.5281/zenodo.6502901)
 
-The echemdb module (i) provides direct access to the database and 
-the entries presented on the [EchemDB Website](https://echemdb.github.io/website) 
-and (ii) allows creating a database from local files which have 
-the same file structure.
+This echemdb module provides a Python library to interact with a database of 
+[frictionless datapackages](https://frictionlessdata.io/)
+containing electrochemical data following [echemdb's metadata schema](https://github.com/echemdb/metadata-schema).
+Such a database can be generated from the data schown on [echemdb.org](https://www.echemdb.org)
```
Comment (saraedum) [inline doc/index.md]:
```suggestion
Such a database can be generated from the data on [echemdb.org](https://www.echemdb.org) 
```

File: doc/index.md [hunk 5]
```diff
@@ -14,20 +14,93 @@ kernelspec:
...
 
-The echemdb module (i) provides direct access to the database and 
-the entries presented on the [EchemDB Website](https://echemdb.github.io/website) 
-and (ii) allows creating a database from local files which have 
-the same file structure.
+This echemdb module provides a Python library to interact with a database of 
+[frictionless datapackages](https://frictionlessdata.io/)
+containing electrochemical data following [echemdb's metadata schema](https://github.com/echemdb/metadata-schema).
+Such a database can be generated from the data schown on [echemdb.org](https://www.echemdb.org) 
+or it can be created from local files which have the same file structure.
```
Comment (saraedum) [inline doc/index.md]:
```suggestion
or from local files.
```

File: doc/index.md [hunk 6]
```diff
@@ -14,20 +14,93 @@ kernelspec:
...
+```
+
+A single entry can be retrieved with the identifiers provided on the website
+(see for example [engstfeld_2018_polycrystalline_17743_f4b_1](https://echemdb.github.io/website/cv/entries/engstfeld_2018_polycrystalline_17743_f4b_1/))
+
+```{code-cell} ipython3
+entry = db['engstfeld_2018_polycrystalline_17743_f4b_1']
+```
+
+Each entry has a set of descriptors which contain information about its `source`, the electrochemical system `system`, or the original publication's figure properties `entry.figure_description`.
```
Comment (saraedum) [inline doc/index.md]:
```suggestion
Each entry has a set of descriptors such as its ``source`` or the electrochemical ``system``.
```

File: doc/index.md [hunk 7]
```diff
@@ -14,20 +14,93 @@ kernelspec:
...
+The data can be visualized in a plotly figure:
+
+```{code-cell} ipython3
+original_entry.plot()
+```
+
+Installation
+=========
+
+The package is hosted on [PiPY](https://pypi.org/project/echemdb/) and can be installed via pip
```
Comment (saraedum) [inline doc/index.md]:
```suggestion
This package is available on [PiPY](https://pypi.org/project/echemdb/) and can be installed with pip:
```

File: doc/index.md [hunk 8]
```diff
@@ -14,20 +14,93 @@ kernelspec:
...
+
+```sh .noeval
+pip install echemdb
+```
+or via conda
```
Comment (saraedum) [inline doc/index.md]:
```suggestion
The package is also available on [conda-forge](https://github.com/conda-forge/echemdb-feedstock) an can be installed with conda:
```

File: doc/index.md [hunk 9]
```diff
@@ -14,20 +14,93 @@ kernelspec:
...
+
+```sh .noeval
+conda install echemdb
```
Comment (saraedum) [inline doc/index.md]:
```suggestion
conda install -c conda-forge echemdb
```

File: doc/index.md [hunk 10]
```diff
@@ -14,20 +14,93 @@ kernelspec:
...
+```
+
+Read the [installation instructions](installation.md) on further details if you want to contribute to the project.
```
Comment (saraedum) [inline doc/index.md]:
```suggestion
See the [installation instructions](installation.md) for further details.
```

File: doc/usage/database_interactions.md [hunk 11]
```diff
@@ -18,7 +18,9 @@ kernelspec:
 
 +++
 
-The entries on [EchemDB Website](https://echemdb.github.io/website) can be downloaded into a local database.
+A database can be generated from a remote or local source.
```
Comment (saraedum) [inline doc/usage/database_interactions.md]:
```suggestion
A database can be generated from a remote or a local source. 
```

File: doc/usage/database_interactions.md [hunk 12]
```diff
@@ -18,7 +18,9 @@ kernelspec:
 
 +++
 
-The entries on [EchemDB Website](https://echemdb.github.io/website) can be downloaded into a local database.
+A database can be generated from a remote or local source. 
+
+In the example the database is created from the available entries shown on [echemdb.org](https://www.echemdb.org).
```
Comment (saraedum) [inline doc/usage/database_interactions.md]:
```suggestion
You can create a database from the entries on [echemdb.org](https://www.echemdb.org):
```

**Review by saraedum — APPROVED** [review]
Thanks. Looks great.

I just had some style suggestions.

I wonder if we should streamline our style of communication (probably we should have a style guide.) Currently, we have a mix of "one can create", "you can create", "we can create", "something can be created". Maybe we should settle for some style and just use it everywhere. (Probably, whatever Python's documentation does?)

Similarly, I wonder if we should replace some of our links with a simpler "[→ Learn more](#)" or something like that. Currently, we also tend to write (overexagerating here) somewhat convoluted "if you would like to see more examples of blablabla, you might want to click on this [link](#)"

Should I create an issue?

**Comment by DunklesArchipel** [comment]
> Thanks. Looks great.
> 
> I just had some style suggestions.
> 
> I wonder if we should streamline our style of communication (probably we should have a style guide.) Currently, we have a mix of "one can create", "you can create", "we can create", "something can be created". Maybe we should settle for some style and just use it everywhere. (Probably, whatever Python's documentation does?)
> 
> Similarly, I wonder if we should replace some of our links with a simpler "[→ Learn more](#)" or something like that. Currently, we also tend to write (overexagerating here) somewhat convoluted "if you would like to see more examples of blablabla, you might want to click on this [link](#)"
> 
> Should I create an issue?

Sounds great. I opened an issue where we can discuss this aspect in more detail.

**Pull request merged**